agent/app/agent.go

```diff
@@ -351,6 +352,16 @@ func (agent *ecsAgent) doStart(containerChangeEventStream *eventstream.EventStre
 		agent.saveMetadata(data.EC2InstanceIDKey, currentEC2InstanceID)
 	}
 
+	// now that we know the container instance ARN, we can build out the doctor
+	// and pass it on to ACS and TACS
+	seelog.Info("Setting up the doctor healthchecks.")
+	doctor, doctorCreateErr := agent.newDoctorWithHealthchecks(agent.cfg.Cluster, agent.containerInstanceARN)
+	if doctorCreateErr != nil {
+		seelog.Warnf("Error starting doctor, healthchecks won't be running: %v", err)
+	} else {
+		seelog.Info("Doctor healthchecks set up properly.")
```

Comment:
We can avoid using adding the two `info` level logs and only keep `warn` level logs for the case when their is error in setting up the Doctor.

Reply:
Ack